src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  --color-blue: #00acdc;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.header__logo-link {
+  display: flex;
+  align-items: center;
+}
+
+.header__nav-list {
+  display: flex;
+  list-style: none;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
```

Comment:
1. For the future, separate `reseting` styles from the rest and move them to the top or in separate `reset.css` file (later). Do the same for all reseting styles.
```
ul {
  list-style: none;
   margin: 0;
   padding: 0;
}
```
2. `justify-content: center;` is redundant